src/core/agent/services/identifierService.ts

```diff
@@ -375,6 +377,82 @@ class IdentifierService extends AgentService {
     });
     return multisigId;
   }
+
+  async rotateMultisig(
+    metadata: IdentifierMetadataRecord,
+    otherIdentifierContacts: ConnectionShortDetails[]
+  ): Promise<string | undefined> {
```

Comment:
@iFergal I think we should store list AID of multisig to use in the feature instead of selecting from UI again

Reply:
Do you mean the multi-sig AIDs or the AIDs/contacts that were used to create that multi-sig AID originally?

Reply:
Yep, exactly

Reply:
Hmm, does KERIA not store any of this info? If not (please fully check this), then we can store the AIDs in the metadata record for the identifier.

Reply:
Refer from Signify-ts integration-test, I see that it is gotten from existed AID, not from Keria, but I will double check with this multisig AID again. If not, will create one more field in metadata.

Reply:
The integration tests in places make certain shortcuts like this because the data is all available in one test so we should investigate further what is available.

But looking at the integration test, it says that each party in the multi-sig rotates their AID out of band and updates their key state. Does this mean if we call a rotation event without doing this, using the existing AIDs that the rotation event will not change any keys?

Reply:
You are right. I tested some case with integration-test and see that:
1. Each party in multi-sig need to rotate their AID before rotating the multi-sig.if not, the rotation with multisig will not change anythings.
2. It has the function "member" the get all members in multi-sig. It provides us the way to get the current OOBI url of each party, so we can resolve it again to get the AID's current state (like the multisig creation did).
Changing this logic...

Reply:
![image](https://github.com/cardano-foundation/cf-identity-wallet/assets/142210850/e79d229b-aa92-4d66-abfc-4f2a114a85cb)

Reply:
OK, for the sake of this core work - lets assume when we call this that all members have rotated - we can implement business logic later to tell others to rotate. I guess we can use the function from your other PR to rotate the identifier used in the multi-sig.

Re "member" - awesome. :)

Reply:
Yep, let me make this work.